.github/workflows/merge.yml

```diff
@@ -0,0 +1,81 @@
+name: merge
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches:
+      - 'v2'
+    tags:
+      - 'v*'
+  workflow_dispatch:
+
+jobs:
+  e2e:
+    name: Build and test
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 15
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [desktop-windows, desktop-macos, desktop-m1]
+        # mode: [plugin, standalone]
+        mode: [plugin]
+    env:
+      GO111MODULE: "on"
+    steps:
+      - name: test-action-unsafe-checkout
+        if: ${{ contains(matrix.os, 'desktop-windows') }}
+        run: |
+          git config --global --add safe.directory C:/actions-runner/_work/compose/compose
```

Comment:
Should we just do this on the runner manually instead of putting it in the job?

(Or maybe there's a way to avoid it all together...?)

Reply:
Ahh, I meant to remove it and see if it was still an issue. I think it should be fine now, given we’ve ran it before (a few times)